Name: Hide NFL App
Version: 1.4.1.1-3
Author: v3ritas
Description: This patch will hide the Sprint NFL App.

--- .orig/usr/palm/applications/com.handson.app.nfl/appinfo.json
+++ /usr/palm/applications/com.handson.app.nfl/appinfo.json
@@ -7,6 +7,7 @@
 	"splashicon": "icon-256x256.png",
 	"version": "1.35.0",
 	"vendor": "Hands-On Mobile",
-	"vendorurl": "http://www.handson.com" 
+	"vendorurl": "http://www.handson.com",
+	"visible": false,
 }
 
